health-services/individual/src/main/resources/db/migration/main/V20221227123000__individual_individual_create_ddl.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE IF NOT EXISTS INDIVIDUAL
+(
+    id                character varying(64),
+    userId            character varying(64),
+    clientReferenceId character varying(64),
+    tenantId          character varying(1000),
+    givenName         character varying(200),
+    familyName        character varying(200),
+    otherNames        character varying(200),
+    -- YYYYMMDD
+    dateOfBirth       character varying(8),
```

Comment:
I'd suggest we use `date` datatype so future operations such as search based on age etc are easier.

Reply:
Done